adofai-roulette.py

```diff
@@ -2,74 +2,50 @@
 
 # setup song, difficulty, and creator lists
 
-tempfile = open("songs.txt", encoding="utf-8")
-songlistbad = tempfile.readlines()
-tempfile2 = open("difficulties.txt", encoding="utf-8")
-difficultylistbad = tempfile2.readlines()
-tempfile3 = open("creators.txt", encoding = "utf-8")
-creatorlistbad = tempfile3.readlines()
+def readLinesAndStrip(filename: str) -> list[str]:
+    with open(filename, encoding="utf-8") as file:
+        lines = file.readlines()
 
-songlist = []
-difficultylist = []
-creatorlist = []
+    return [line.strip() for line in lines]
 
-for a in songlistbad:
-    songlist.append(a.strip())
-for a in difficultylistbad:
-    difficultylist.append(a.strip())
-for a in creatorlistbad:
-    creatorlist.append(a.strip())
+
+songs = readLinesAndStrip("songs.txt")
+difficulties = readLinesAndStrip("difficulties.txt")
+creators = readLinesAndStrip("creators.txt")

```

Comment:
Note the code duplication: when you do the same thing over and over, you wanna make a function, that way you have less code to maintain, and less chance of errors.